doc/md/base/index.md

```diff
@@ -1,6 +1,6 @@
 # Index
 
-* [Array](Array.md) Functions on Arrays
+* [Array](Array.md) Provides extended utility functions on Arrays. Note the difference between
```

Comment:
Why is this cut off?

Reply:
No idea. mo-doc broken?

Reply:
This is the generated index.md. It must be truncating the first line.
The Array.md looks fine.
I can't live with this for now...
I'll fix it by hand for this release.

Reply:
Done

Reply:
Ah okay. So the first line of documentation gets pulled as the summary?

I'll fix that line in `Array.mo`

Reply:
https://github.com/dfinity/motoko-base/pull/440

Reply:
Beat you to it.

Reply:
Nvm you got it

Reply:
@kentosugama can you approve? I'm hoping all the merges will happen by morning so I can release 0.7.x

Reply:
Stamped